src/data/option/language_options.tsx

```diff
@@ -0,0 +1 @@
+export const optionsLanguage = ['English', 'German', 'Polish'];
```

Comment:
na to nie trzeba nego pliku. Options trzymaj sobie w komponencie tak gdzie je wykorzystasz. 
Ewentualnie jak już chcesz nowy plik to obok danego komponentu. Moim zdaniem bez sensu nowy folder gdzieś na to robić.

Reply:
[value: 'en', label: 'English', ...]